service/history/replication/stream_sender.go

```diff
@@ -358,6 +360,10 @@ Loop:
 		if err != nil {
 			return err
 		}
+		err = s.flowController.Wait(enumsspb.TASK_PRIORITY_UNSPECIFIED)
```

Comment:
task priority?

Reply:
yeah, I am planning to merge this PR after the the tiered sender implementation. So here is mostly for demoing how to use the wait. After the tiered sender PR merged, this part need to be fixed.